pysparkle/pysparkle/data_quality/data_quality.py

```diff
@@ -0,0 +1,28 @@
+from pyspark.sql import SparkSession
+
+from pysparkle.adls_utils import set_env, set_spark_properties
+from pysparkle.data_quality.data_quality_utils import create_datasource_config, create_expectation_suite, \
+execute_validations
+
+def data_quality_main(DQ_conf):
+    dataset_name = DQ_conf["dataset_name"]
+    print("Running Data Quality processing...")
+    spark = SparkSession.builder.appName(
+        f'Data Quality {DQ_conf["container_name"]} {dataset_name}'
```

Comment:
Can you make the application name without whitespaces?
```suggestion
        f'DataQuality-{DQ_conf["container_name"]}-{dataset_name}'
```